opam

```diff
@@ -22,4 +22,4 @@ depexts: [
   [["debian"] ["mpi-default-dev"]]
   [["ubuntu"] ["mpi-default-dev"]]
 ]
-
+available: [[ ocaml-version >= "4.03" ]]
```

Comment:
Is is possible to support 4.02.3 as a minimum (base version of OCaml on Debian stable https://packages.debian.org/stretch/ocaml)?

Reply:
Changed!